source/MRMesh/MRChangeLabelAction.h

```diff
@@ -5,6 +5,7 @@
 #include "MRHistoryAction.h"
 #include "MRPositionedText.h"
 #include "MRVector2.h"
+#include "MRMesh/MRObjectLabel.h"
```

Comment:
can we avoid it at least for not-binding mode?

Reply:
Maybe we can remove this, but it's impossible to use this class without this include, so I think keeping it is okay. This code dereferences a `shared_ptr<ObjectLabel>`.

Reply:
Better remove please

Reply:
Per discussion in Fleep, agreed to not remove.